locales/vi-VN/translation.json

```diff
@@ -163,13 +163,13 @@
   "Enter password": "Nhập mật khẩu",
   "Login": "Đăng nhập",
   "Security Check": "Kiểm tra bảo mật",
-  "Alright! Time to check if you got your words right!": "Tuyệt! Bây giờ hãy kiểm tra nếu bạn ghi chính xác các cụm từ nhé!",
-  "Select the words in the right order. Ready?": "Chọn các từ khôi phục với trình tự chính xác. Sẳn sàng chứ?",
-  "Ready!": "Sẳn sàng!",
+  "Alright! Time to check if you got your words right!": "Tuyệt vời! Bây giờ hãy kiểm tra nếu bạn ghi chính xác các cụm từ nhé!",
+  "Select the words in the right order. Ready?": "Chọn các từ khôi phục với thứ tự chính xác. Sẳn sàng chứ?",
+  "Ready!": "BẮT ĐẦU!",
```

Comment:
Does it need to be all in capital?

Reply:
My typo, thank you. 
@vynguyen94 I hope you and I could spend time on checking typos like the same way that @nop33 advices us for the next time. 

Thanks in advanced.